.github/workflows/e2e.yml

```diff
@@ -0,0 +1,63 @@
+# this workflow will run on every pr to make sure the project is following the guidelines
+
+# after labeler, run other actions with strict permissions
+name: CI
+
+on:
+  workflow_run:
+    workflows: [labeler]
```

Comment:
should this be uppercase?

Reply:
no idea. is it the workflow name or the job name